packages/react-table/src/demos/table-demos/StickyFirstColumn.jsx

```diff
@@ -0,0 +1,204 @@
+  const getSortableRowValues = fact => {
+    const {
+      name,
+      state,
+      detail1,
+      detail2,
+      detail3,
+      detail4,
+      detail5,
+      detail6,
+      detail7,
+      detail8,
+      detail9,
+      detail10,
+      detail11,
+      detail12,
+      detail13,
+      detail14
+    } = fact;
+    return [
+      name,
+      state,
+      detail1,
+      detail2,
+      detail3,
+      detail4,
+      detail5,
+      detail6,
+      detail7,
+      detail8,
+      detail9,
+      detail10,
+      detail11,
+      detail12,
+      detail13,
+      detail14
+    ];
+  };
+  let sortedFacts = facts;
+  if (activeSortIndex !== null) {
+    sortedFacts = facts.sort((a, b) => {
+      const aValue = getSortableRowValues(a)[activeSortIndex];
+      const bValue = getSortableRowValues(b)[activeSortIndex];
+      if (aValue === bValue) {
+        return 0;
+      }
+      if (activeSortDirection === 'asc') {
+        return aValue > bValue ? 1 : -1;
+      } else {
+        return bValue > aValue ? 1 : -1;
+      }
+    });
+  }
+  const getSortParams = columnIndex => ({
+    sortBy: {
+      index: activeSortIndex,
+      direction: activeSortDirection
+    },
+    onSort: (_event, index, direction) => {
+      setActiveSortIndex(index);
+      setActiveSortDirection(direction);
+    },
+    columnIndex
+  });
+  return (
+    <DashboardWrapper hasPageTemplateTitle>
+      <PageSection isWidthLimited padding={{ default: 'noPadding', xl: 'padding' }}>
+        <Card component="div">
+          <InnerScrollContainer>
+            <TableComposable aria-label="Sticky column table" gridBreakPoint="">
+              <Thead>
+                <Tr>
+                  <Th isStickyColumn stickyMinWidth="100px" hasRightBorder modifier="truncate" sort={getSortParams(0)}>
+                    {columnNames.name}
+                  </Th>
+                  <Th modifier="truncate" sort={getSortParams(1)}>
+                    {columnNames.state}
+                  </Th>
+                  <Th modifier="truncate">{columnNames.header3}</Th>
+                  <Th modifier="truncate">{columnNames.header4}</Th>
+                  <Th modifier="truncate">{columnNames.header5}</Th>
+                  <Th modifier="truncate">{columnNames.header6}</Th>
+                  <Th modifier="truncate">{columnNames.header7}</Th>
+                  <Th modifier="truncate">{columnNames.header8}</Th>
+                  <Th modifier="truncate">{columnNames.header9}</Th>
+                  <Th modifier="truncate">{columnNames.header10}</Th>
+                  <Th modifier="truncate">{columnNames.header11}</Th>
+                  <Th modifier="truncate">{columnNames.header12}</Th>
+                  <Th modifier="truncate">{columnNames.header13}</Th>
+                  <Th modifier="truncate">{columnNames.header14}</Th>
+                  <Th modifier="truncate">{columnNames.header15}</Th>
+                  <Th modifier="truncate">{columnNames.header16}</Th>
```

Comment:
`thead` cells are set to truncate by default, so you don't need `modifier="truncate"`. I'm not sure why the core demo has that, but it needs to be updated. I've opened [4882](https://github.com/patternfly/patternfly/issues/4882) to do that.  I'll set the first two `th` cells to `pf-m-fit-content` and remove the subsequent `pf-m-truncate` mods, so you can match that pattern.